nsv13/code/modules/overmap/types.dm

```diff
@@ -55,6 +55,19 @@
 	max_integrity = 1500 //Max health
 	integrity_failure = 1500
 
+/obj/structure/overmap/nanotrasen/battleship
+	name = "judgement class battlestar"
+	desc = "A gigantic battleship capable of pummelling entire enemy fleets into the ground with its advanced firepower. Ships like these are extremely expensive to produce, and are thus rarely seen in actual combat these days."
+	icon = 'nsv13/icons/overmap/nanotrasen/battleship.dmi'
+	icon_state = "battleship"
+	mass = MASS_TITAN
```

Comment:
I'd change this to MASS_LARGE because piloting this thing isn't fun at all.

Reply:
With the weaponry that GALACTICA can bring to bear I think it's better balanced remaining titan class.